### ISSUE TITLE
Filter GraphQL Logging to exclude IntrospectionQueries in development

### ISSUE THREAD

**Issue by dthyresson** [redwoodjs/redwood#3251]
@KrisCoulson asked in Discord if it was possible to filter GraphQL logging in developer to exclude queries made by the GraphQL Playground Introspection queries, such as:

![image](https://user-images.githubusercontent.com/1051633/130069432-68756867-beab-4c5c-9e47-ac1fe2f254da.png)

Since the playground polls frequently, this issues numerous log statements.

However, the playground is sending "normal" queries -- they are not a different type of query.

The are identified by its operation name "IntrospectionQuery" which could be used to filter out.

Perhaps the `operationName` could be a boolean (include the name or not) or some structure of include or exclude names like:

```js
operationName: true
```

or 

```js
operationName: { exclude: 'IntrospectionQuery' }
```

So, that would mean, yes, add the operation name to the out log statement, but if the name was in the exclusion set, don't log and in `onExecuteDone` of the `useRedwoodLogger` return `undefined` id should not log.

**Comment by dthyresson** [comment]
Also, @KrisCoulson I believe there is a setting to increase the playground's polling interval:

```
 "schema.polling.interval": 2000,
```

Just go the Settings (the gear icon)

![image](https://user-images.githubusercontent.com/1051633/130071986-d5029bad-0858-4c08-9a58-c1a0ff7fa711.png)


and then change to a much longer interval and that will reduce the log output because there will be fewer calls made

**Comment by callingmedic911** [comment]
Hey @dthyresson! I was checking this issue and wanted to ask that even if we filter out logs based on `operationName`, [`morgan`](https://github.com/redwoodjs/redwood/blob/main/packages/api-server/src/app.ts#L10) would still print the path and other details like `POST /graphql 200 4.567 ms - 23386`. Logs would much be less but it could possibly still spam the output.

I think some trace that a request reached the server should always be there - which `morgan` is doing. What do you think?

**Comment by callingmedic911** [comment]
@dthyresson Gentle reminder DT. I'd be glad to work on this.

**Comment by dthyresson** [comment]
@callingmedic911 Hi. I'm really not sure if we need to implement this filter -- by setting the polling interval or less frequent (or even completely off) that limits the requests made by the playground in dev and not sure the logic step is needed to check and screen out these.

But, if you are interested, let's see what a filter could look like. Give it a god if this interests you. Thanks!

**Comment by callingmedic911** [comment]
> But, if you are interested, let's see what a filter could look like. Give it a god if this interests you. Thanks!

Yeah it's not completely filtering out (because of morgan). I tried something earlier. And also, it could be confusing if someone deployed with excluded option to prod that was meant only for dev env.

Anyways, let me raise a draft PR to show you.
